### ISSUE TITLE
Open virtual port with specific baudrate

### ISSUE THREAD

**Issue by iPAS** [ezramorris/PyVirtualSerialPorts#1]
Hi. This is a good work. I love it. However, to emulate the link between embedded device, I need to be able to determine the baudrate of the virtual ports. Do you have a plan or a guide of implementing such function ?

**Comment by ezramorris** [comment]
Hi,

Thanks for raising this issue.

Please can you give some more detail of the use case for this? This is implemented using Linux PTYs, which aren't connected to any physical device, so baud rate isn't really a concept, i.e. you should be able to open the ports with any baud rate.

Many thanks.

**Comment by iPAS** [comment]
I tried to sniff the data between two embedded devices without using tapping wires. So, I connected the first one, through UART, on USB dongle, and, connected the second one with another USB dongle on the same machine. 
Then, those two devices appeared as the device files: /dev/ttyUSB0 and /dev/ttyUSB1 respectively.

![2022-12-22_10-18](https://user-images.githubusercontent.com/1320767/209048687-4c516b41-870d-4d3b-a0ec-ffe221e4f706.png)

At glance, the PyVirtualSerialPorts that I mean, is the one that can capture the device files as well.
However, finally I realized that I should implement the 'blue' arrow lines by my self. It is not the function in purpose of the library any more.
